zcash_client_sqlite/src/testing.rs

```diff
@@ -1938,22 +1937,62 @@ impl BlockCache {
     }
 }
 
+pub(crate) struct NoteCommitments {
+    sapling_note_commitments: Vec<sapling::Node>,
+    #[cfg(feature = "orchard")]
+    orchard_note_commitments: Vec<MerkleHashOrchard>,
+}
+
+impl NoteCommitments {
+    #[allow(dead_code)]
+    pub(crate) fn sapling_note_commitments(&self) -> &[sapling::Node] {
```

Comment:
The type is already called `NoteCommitments`, so just name this method `NoteCommitments::sapling`:
```suggestion
    pub(crate) fn sapling(&self) -> &[sapling::Node] {
```